datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/jmx_metrics.py

```diff
@@ -0,0 +1,69 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from collections import defaultdict
+
+import click
+import yaml
+
+from datadog_checks.dev.utils import read_file
+
+from ...utils import get_jmx_metrics_file, get_valid_integrations, is_jmx_integration
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
+
+
+@click.command('jmx-metrics', context_settings=CONTEXT_SETTINGS, short_help='Validate JMX metrics files')
+@click.option('--verbose', '-v', is_flag=True, help='Verbose mode')
+def jmx_metrics(verbose):
+    """Validate all default JMX metrics definitions."""
+
+    echo_info("Validating all JMX metrics files...")
+
+    saved_errors = defaultdict(list)
+    integrations = sorted(check for check in get_valid_integrations() if is_jmx_integration(check))
+    for check_name in integrations:
+        jmx_metrics_file, file_exists = get_jmx_metrics_file(check_name)
+
+        if not file_exists:
+            saved_errors[check_name].append(f'{jmx_metrics_file} does not exist')
+            continue
+
+        jmx_metrics_data = yaml.safe_load(read_file(jmx_metrics_file)).get('jmx_metrics')
```

Comment:
Should we also verify that if the jmx integration uses spec.yaml, we should check that the jmx templates are included? We recently added something similar for http wrapper validation too.

Reply:
@ChristineTChen Good idea. I added a validation for the config spec.

It's done a bit differently than http wrapper config spec validation. cc @yzhan289